src/core/createGetLinkElementDetails.js

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line import/no-named-as-default
+import createGetLinkDetails from "../components/ActivityCollector/createGetLinkDetails";
+
+export default (window, config, logger) => {
```

Comment:
I'd prefer to pass what's needed directly (onBeforeLinkClickSend) instead of the whole config object.